### ISSUE TITLE
how to import audiovideoconfiguration and image?

### ISSUE THREAD

**Issue by ichenli** [palanjyan/onvif-videotech-python#1]
hi, as a newcomer to onvif, I don`t know how to successfully import audiovideoconfiguration and image when I run control.py. Could u do me a favor? Thanks

**Comment by palanjyan** [comment]
Hi, you can't install the python-onvif-zeep library or specifically my code?
If the second use it in control.py
```
from onvif import ONVIFCamera
from audioVideoConfigurations import AudioConfigurations, VideoConfigurations

camera = ONVIFCamera('192.168.14.14', 80, 'admin', 'pass', 'wsdl/', no_cache=False)

media = camera.create_media_service()
profile = media.GetProfiles()[0]

#audio = AudioConfigurations(media, profile)
#audio.encoding("AAC")

video = VideoConfigurations(media, profile)
video.bitrateLimit(3000)
```

**Comment by ichenli** [comment]
I fixed it !!!
At the beginning, I failed to install python-onvif-zeep, after successfully installing, your code works.
At the same time , I also have a question: To install python-onvif-zeep, I have to install the dependence(zeep), it really time consuming, is that normal?

**Comment by palanjyan** [comment]
@ichenli how long will it take?

**Comment by palanjyan** [comment]
@ichenli zeep is a powerful enough SOAP client, but I do not think it should be installed for a long time

**Comment by ichenli** [comment]
To install python-onvif-zeep, I installed zeep, but throw the error: "command 'x86_64-linux-gnu-gcc' failed with exit status 1. " Then followed by the "https://blog.csdn.net/wang1144/article/details/42277179" , I fixed it. In this process, "pip install lxml" cost about 5 mins.
Then I run "cd python-onvif-zeep && python setup.py install", it also cost about 5mins at the last step of   install lxml. Maybe the network is not good enough.
Anyway , thank you for your code and your answers. It really helps me a lot !!!
